### ISSUE TITLE
[Bug Report] El Datepicker - focusing datepicker input with pressing tab button makes datepicker to hide after clicking next/prev month/year

### ISSUE THREAD

**Issue by MikeSaprykin** [element-plus/element-plus#1809]
### Element Plus version
1.0.2-beta.39

### OS/Browsers version
macOs Catalina, Chrome 89, Windows 10 Edge 89

### Vue version
3.0.11

### Reproduction Link
https://codesandbox.io/s/ecstatic-elbakyan-1eny5?fontsize=14&hidenavigation=1&theme=dark

### Steps to reproduce
1. Focus basic input with mouse or tab
2. Press tab to focus datepicker input
3. When datepicker opens, try to select previous month by clicking previous month button
4. When datepicker is closed - focus datepicker input with mouse
5. Try to select previous month by clicking previous month button

### What is Expected?
After focus with tab and interaction with datepicker ( choosing next/prev year/month ) datepicker stays opened.

### What is actually happening?
After focus with tab and interaction with datepicker ( choosing next/prev year/month ) datepicker is closed.

If you open datepicker by clicking on input and then try to change month/year - everything works like expected. The behaviour of datepicker popper is not consistent between focus by pressing tab and focus by mouse event

This behaviour can be seen on element plus datepicker docs page, while can't be reproduced with old, vue 2 element ui

**Comment by MikeSaprykin** [comment]
It seems like this issue is related to click outside directive.

Commenting out line 352 in file `/packages/time-picker/src/common/picker.vue` in function onClickOutside seems to fix the issue (but of course removes ability to close picker on click outside)

```
const onClickOutside = () => {
   if (!pickerVisible.value) return
   // pickerVisible.value = false
}
```

**Comment by MikeSaprykin** [comment]
Digging into issue a little bit more what I found out so far:

The issue is indeed in click outside directive.

So in `packages/directives/click-outside/index.ts`

line 47

```
const popperRef = (binding.instance as ComponentPublicInstance<{
      popperRef: Nullable<HTMLElement>
    }>).popperRef
```

The problem is that in picker.vue component there is no `popperRef` property.
but there is `refPopper` which is a ref to popper element with `popperRef` property

I've investigated a bit how it works in select or filter-panel

It seems like in picker.vue this computed property also should be added

```
   const popperPaneRef = computed(() => {
      return refPopper.value?.popperRef
    })
```

and used in click outside directive

```
v-clickoutside:[popperPaneRef]="onClickOutside"
```

**Comment by MikeSaprykin** [comment]
This indeed fixed the issue, I can create a PR with this fix

**Comment by JeremyWuuuuu** [comment]
Closing since its fixed in #1812
